xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -1146,7 +1148,15 @@ void CVideoLibrary::UpdateVideoTag(const CVariant &parameterObject, CVideoInfoTa
   if (ParameterNotNull(parameterObject, "mpaa"))
     details.SetMPAARating(parameterObject["mpaa"].asString());
   if (ParameterNotNull(parameterObject, "imdbnumber"))
-    details.SetIMDBNumber(parameterObject["imdbnumber"].asString());
+    details.SetUniqueID(parameterObject["imdbnumber"].asString());
+  if (ParameterNotNull(parameterObject, "uniqueid"))
+  {
+    std::map<std::string, std::string> uniqueids;
+    for (CVariant::const_iterator_map rIt = parameterObject["uniqueid"].begin_map(); rIt != parameterObject["uniqueid"].end_map(); rIt++)
```

Comment:
Even though this code is most likely obsolete because it needs to be aligned with how artwork is handled this `for` loop could be replaced with a simple
```
std::map<std::string, std::string> uniqueids(parameterObject["uniqueid"].begin_map(), parameterObject["uniqueid"].end_map());
```